### DISCUSSION TITLE
Slack link Problem on GSoC Community Page!

### DISCUSSION THREAD

**Discussion by sahilempire** [orcasound/.github#7] [Q&A]
The link given on the Community Page of GSoC `https://summerofcode.withgoogle.com/programs/2022/organizations/orcasound` is not working correctly, as I want to work with your Organisation, And I want to discuss something with the maintainers, but because of the link error, I can't do that.

**Comment by scottveirs** [comment]
[Try this Slack invitation](https://join.slack.com/t/orcasound/shared_invite/zt-1jr7te193-ObLYcTxx4s7SO7EA9kmcaw) instead, @sahilempire

**Comment by scottveirs** [comment]
This was resolved via #8 so I'll lock this conversation now.
